src/battle_script_commands.c

```diff
@@ -3434,26 +3650,36 @@ static void Cmd_getexp(void)
                 if (holdEffect == HOLD_EFFECT_EXP_SHARE)
                     viaExpShare++;
             }
+            #if (B_SCALED_EXP == GEN_5) && (B_SCALED_EXP != GEN_6)
```

Comment:
I'm not sure if this should be `>=` instead since later Gens should be valid values and the check for Gen 6 as a result feels redundant since only Gen 5 is the valid value for this if block to compile.

Reply:
Should be `>=`, yes. This is what happens when you replace stuff and don't think.